wd_demo.au3

```diff
@@ -458,6 +458,7 @@ Func _USER_WD_Sleep($iDelay)
 EndFunc   ;==>_USER_WD_Sleep
 
 Func SetupGecko()
+	_WD_UpdateDriver('firefox', @ScriptDir , False)
```

Comment:
Not sue why you specify False for the 3rd parameter when this is the default value.

Reply:
ok, will support all yours recomendation which you post in above comments.

Reply:
Resolved ?